presto-accumulo/src/test/java/com/facebook/presto/accumulo/index/TestIndexer.java

```diff
@@ -0,0 +1,354 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.accumulo.index;
+
+import com.facebook.presto.accumulo.metadata.AccumuloTable;
+import com.facebook.presto.accumulo.model.AccumuloColumnHandle;
+import com.facebook.presto.accumulo.serializers.AccumuloRowSerializer;
+import com.facebook.presto.accumulo.serializers.LexicoderRowSerializer;
+import com.facebook.presto.spi.type.Type;
+import com.facebook.presto.type.ArrayType;
+import com.google.common.collect.ImmutableList;
+import org.apache.accumulo.core.client.BatchWriterConfig;
+import org.apache.accumulo.core.client.Connector;
+import org.apache.accumulo.core.client.Instance;
+import org.apache.accumulo.core.client.IteratorSetting;
+import org.apache.accumulo.core.client.Scanner;
+import org.apache.accumulo.core.client.mock.MockInstance;
+import org.apache.accumulo.core.client.security.tokens.PasswordToken;
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.Mutation;
+import org.apache.accumulo.core.data.Range;
+import org.apache.accumulo.core.data.Value;
+import org.apache.accumulo.core.security.Authorizations;
+import org.apache.accumulo.core.security.ColumnVisibility;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Iterator;
+import java.util.Map.Entry;
+import java.util.Optional;
+
+import static com.facebook.presto.spi.type.BigintType.BIGINT;
+import static com.facebook.presto.spi.type.VarcharType.VARCHAR;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+
+public class TestIndexer
+{
+    private static final LexicoderRowSerializer SERIALIZER = new LexicoderRowSerializer();
+
+    private static byte[] encode(Type type, Object v)
+    {
+        return SERIALIZER.encode(type, v);
+    }
+
+    private static final byte[] AGE = b("age");
+    private static final byte[] CF = b("cf");
+    private static final byte[] FIRSTNAME = b("firstname");
+    private static final byte[] SENDERS = b("arr");
+
+    private static final byte[] M1_ROWID = encode(VARCHAR, "row1");
+    private static final byte[] AGE_VALUE = encode(BIGINT, 27L);
+    private static final byte[] M1_FNAME_VALUE = encode(VARCHAR, "alice");
+    private static final byte[] M1_ARR_VALUE = encode(new ArrayType(VARCHAR), AccumuloRowSerializer
```

Comment:
Remove most/all of the wrapping in this file. It seems to be wrapped to an arbitrary width and makes the code harder to read.